packages/tpetra/core/test/MultiVector/CMakeLists.txt

```diff
@@ -106,3 +106,14 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   NUM_MPI_PROCS 1
   STANDARD_PASS_OUTPUT
   )
+
+TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  MultiVector_MicroBenchmark
+  SOURCES
+    MultiVector_MicroBenchmark
+    ${TEUCHOS_STD_UNIT_TEST_MAIN}
+  COMM serial mpi
+  NUM_MPI_PROCS 1
+  STANDARD_PASS_OUTPUT
+  ${MAXNP}
```

Comment:
I'm pretty sure you can delete the `${MAXNP}` line -- that's left over from some Chris Baker - era test machine.